scripts/lerna-ttsc.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/env bash
+
+if [ "${LERNA_ROOT_PATH}" != "" ]; then
+  PACKAGE_DIR=$(pwd)
+  cd ${LERNA_ROOT_PATH}
+  if [[ $# == 0 ]]; then
+    ttsc --project ${PACKAGE_DIR}/tsconfig.json
+  else
+    ttsc $@
```

Comment:
This should allow you to still do something like `../../../../scripts/lerna-ttsc.sh --project $(pwd)/other.tsconfig.json`